acceptance/setup/pre_suite/40_install_deps.rb

```diff
@@ -47,7 +47,11 @@
       if test_config[:install_type] == :git then
         case os
         when :debian
-          on database, "apt-get install -y --force-yes openjdk-7-jre-headless rake unzip"
+          if database['platform'].start_with?('debian-8-')
```

Comment:
here you're using `debian-8-` but above you're using `debian-8` (probably not important, but figured I might as well point it out).